grpc/src/test/java/com/linecorp/armeria/common/grpc/GrpcExceptionHandlerFunctionBuilderTest.java

```diff
@@ -111,21 +111,25 @@ void mapStatus() {
                 GrpcExceptionHandlerFunction
                         .builder()
                         .on(A2Exception.class, (ctx, throwable, metadata) -> Status.PERMISSION_DENIED)
+                        .on(A1Exception.class, (ctx1, cause, metadata) -> Status.DEADLINE_EXCEEDED)
                         .build();
 
         for (Throwable ex : ImmutableList.of(new A2Exception(), new A3Exception())) {
-            final Status status = Status.UNKNOWN.withCause(ex);
             final Metadata metadata = new Metadata();
-            final Status newStatus = GrpcStatus.fromExceptionHandler(exceptionHandler, ctx, status, metadata);
+            final Status newStatus = exceptionHandler.orElse(GrpcExceptionHandlerFunction.ofDefault())
+                                                     .apply(ctx, ex, metadata);
             assertThat(newStatus.getCode()).isEqualTo(Code.PERMISSION_DENIED);
             assertThat(newStatus.getCause()).isEqualTo(ex);
             assertThat(metadata.keys()).isEmpty();
         }
 
-        final Status status = Status.DEADLINE_EXCEEDED.withCause(new A1Exception());
+        final A1Exception cause = new A1Exception();
         final Metadata metadata = new Metadata();
-        final Status newStatus = GrpcStatus.fromExceptionHandler(exceptionHandler, ctx, status, metadata);
-        assertThat(newStatus).isSameAs(status);
+        final Status newStatus = exceptionHandler.orElse(GrpcExceptionHandlerFunction.ofDefault())
+                                                 .apply(ctx, cause, metadata);
```

Comment:
Ditto
```suggestion
        final Status newStatus = exceptionHandler.apply(ctx, cause, metadata);
```

Reply:
Applied feedback, Thanks.
I've checked that test is working fine after change.